### DISCUSSION TITLE
Archiving large data

### DISCUSSION THREAD

**Discussion by Gitt-hubb** [Yutaka-Sawada/MultiPar#123] [Q&A] [answered]
Hi 👋,  previously i asked questions by par3 development for windows. Reason was I'm creating pars for 4.5tb data all video files ranging 350mb - 1.5gb size.

I'm new to creating recovery par 😳...

For 1TB roughly 300GB is assigned
It pretty much used the max source block

![image](https://github.com/Yutaka-Sawada/MultiPar/assets/157591827/fad11f71-e9bc-4062-8efc-c9f865a01c7d)

![image](https://github.com/Yutaka-Sawada/MultiPar/assets/157591827/d95328a2-00fb-4a77-b86d-abee11944610)

From recovery point of view is this look ok ? 
Anthing else i need to know or improve.
Or should i just wait until par3 beta get released.

**Comment by Yutaka-Sawada** [comment, marked as answer]
> From recovery point of view is this look ok ?

No, it's not ok. Parchive technology makes recovery data for partial loss. It requires whole available data to repair damaged files. If you create PAR2 files from 4000 source files, you will need them to repair even a very small file. It will be slow verification and require long time to repair.

You should consider possible damage rate. Parchive is used mostly for small random error. If many files will be lost at once in future, you would better take backup instead of creating PAR2 files. After backup many files, use parchive to protect and strengthen the backuped files.

Also, you should grouping you source files. Create PAR2 files for each group of source files. Such like, category or date. You may make folders for categorize your video. Then, create PAR2 files for each folder is good. I did so for my MP3 music files.

When whole files in a folder is lost in future, you will be able to restore them from backup. Use PAR2 files to confirm the completeness of their contents. If a few files are lost in a folder, you may recover them from PAR2 files. This method is much faster.

I want to say two points below.
1) If you want to set large redundancy, consider to backup them mainly.
2) When there are many source files, don't try to protect them at once.

**Reply by Gitt-hubb** [reply]
Yes 😀 i have backup them as well one additional copy for whole data, as well as creating par 300gb per 1 tb. Then burn these par to cold medium such as bluray bdxl.✨️
My concern is should i wait for mutipar3 to come out or this multipar2 should be alright🤔
